language/bytecode-verifier/src/borrow_graph.rs

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//! This module defines the (acyclic) borrow graph for the type and memory safety analysis.
+use crate::nonce::Nonce;
+use mirai_annotations::{checked_postcondition, checked_precondition};
+use std::collections::{BTreeMap, BTreeSet};
+
+/// There are two types of edges in the borrow graph.
+/// A strong edge carries more information than a weak edge.
```

Comment:
Strong and weak are pretty loaded terms when it comes to program analysis. In this case "more information" does not really provide the reader with more information. What information? When do you have more information and when not? Why?

Reply:
Instead of providing piecemeal explanation on various components of borrow graph, I will provide a detailed explanation of the borrow graph concepts in one comment at the top of the file.  That should help.